cluster/manager/manager.go

```diff
@@ -76,6 +76,26 @@ type ClusterManager struct {
 	snapshotPrefixes   []string
 }
 
+// ErrNewNodeOverMaxCapacity used when a new node (not yet part of the cluster) triggers max #nodes capacity.
```

Comment:
This is fine. Please add some documentation on how to use it by a caller.

Reply:
@lpabon -- please check follow-up 93f0e8d